src/ui/PromotedResultsBadge/PromotedResultsBadge.ts

```diff
@@ -18,13 +18,12 @@ export interface IPromotedResultsBadgeOptions {
 }
 
 /**
- * The `PromotedResultsBadge` component adds a badge to promoted results in your interface.
+ * Depending on its configuration, this component will render badges on query result items whose ranking scores were increased by [featured results](https://docs.coveo.com/en/1961/) query pipeline rules and/or [Coveo ML ART](https://docs.coveo.com/en/1671/#automatic-relevance-tuning-art-feature).
  *
- * To be considered promoted, a result needs to either:
- * - be a Featured Result configured through a Coveo Query Pipeline (see [Adding and Managing Query Pipeline Featured Results](https://docs.coveo.com/en/1961/))
- * - be a recommended result by Coveo Machine Learning (see [Automatic Relevance Tuning (ART) Feature](https://docs.coveo.com/en/1671/#automatic-relevance-tuning-art-feature)).
+ * This component can be put anywhere in the markup configuration of a search interface. However, it is meant to be initialized only once, and should thus never be included in a result template.
```

Comment:
This is the line that addresses the "issue" described in https://coveord.atlassian.net/browse/JSUI-2774.

I took the opportunity to update/improve the entire component's documentation at the same time :)